### PR TITLE
Update the 'offline' page

### PR DESCRIPTION
**Pull request by ganicke**
Start of a discussion on what to add for this:

* Reconnect is sometimes necessary when packages are added?
* The 'Native Apps' section, I ditched it. What about Windows 10 apps for certain targets? Have an override section to include a mention of those?
* Are there concerns now about app cache constraints?
* Serving locally, keep this section but make it more friendly?

### CODE REVIEW

**Comment by pelikhan** [comment]
Wrong branch...

**Comment by ganicke** [comment]
Since the update idea came from the microbit page, I started in `v0`. Doesn't microbit get it's common-docs from this branch?

File: common-docs/offline.md [hunk 1]
```diff
@@ -2,15 +2,10 @@
 
 ## Web application
 
-**@homeurl@ is an HTML5 web application** that automatically gets cached locally by your browser. 
-Once the web app is loaded and you have compiled at least once, you will have all the code needed to work without an internet connection.
+**[MakeCode](@homeurl@)** is an **HTML5 web application** that's automatically cached locally when first viewed in your browser. After the web app is loaded and once the code blocks have compiled for the **first** time, you will have everything you need to continue working without an internet connection. This is true for the all of the blocks you see in the editor when it's first loaded. If you decide to add a [package](/packages), it's possible that you'll again need to connect to the internet to allow the code in the new package to compile.
```
[the remark this answers is not in the document]
Comment (pelikhan) [reply common-docs/offline.md]: Also, the default .hex file is now part of the manifest so you don't need to compile once to get it to be fully usaeable.

File: common-docs/offline.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Web application
 
-**[MakeCode](@homeurl@)** is an **HTML5 web application** that's automatically cached locally when first viewed in your browser. After the web app is loaded and once the code blocks have compiled for the **first** time, you will have everything you need to continue working without an internet connection. This is true for the all of the blocks you see in the editor when it's first loaded. If you decide to add a [package](/packages), it's possible that you'll again need to connect to the internet to allow the code in the new package to compile.
```
Comment (ganicke) [inline common-docs/offline.md]: This is no longer a v0 requirement either?

File: common-docs/offline.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
+**[MakeCode](@homeurl@)** is an **HTML5 web application** that's automatically cached locally (saved to your computer or device) when first viewed in your browser. After the web app has loaded you will have everything you need to continue working without an internet connection. If you decide to add a [package](/packages), it's possible that you'll again need to connect to the internet to allow the code in the new package to compile.
```
Comment (ganicke) [inline common-docs/offline.md]: Is it true that adding a package needs a reconnect to the mothership? Well, yes to at least add the package, but a compile too?

File: common-docs/offline.md [hunk 4]
```diff
@@ -16,6 +16,6 @@ When running on actual hardware, the code you write in JavaScript is compiled in
 
 ## #target-app
 
-## Hosting MakeCode locally
+## Hosting MakeCode locally #local-serve
 
-More experienced users can download the entire [PXT Toolchain](https://github.com/Microsoft/pxt) and use the [command line interface](/cli) (CLI) to compile and deploy your scripts locally. PXT provides a great out-of-the-box experience using [Visual Studio Code](/code), a lightweight cross-platform code editor. See the @githubUrl@ project page for instructions on setting up a MakeCode local server.
+The open source editions of MakeCode can be served locally on your own computer. More experienced users can download the entire [PXT Toolchain](https://github.com/Microsoft/pxt) and use the [command line interface](/cli) (CLI) to compile and deploy scripts locally. PXT provides a great out-of-the-box experience when used with [Visual Studio Code](/code), a lightweight cross-platform code editor. See the @githubUrl@ project page for instructions on setting up a MakeCode local server.
```
Comment (ganicke) [inline common-docs/offline.md]: Is this the right way to qualify when you can local serve?

**Pull request merged**